README.md

```diff
@@ -84,170 +98,172 @@ This code will generate the following terminal output on success:
 
-### Dashboard 
+#### Language
 
-Speech Center features a dashboard at https://dashboard.speechcenter.verbio.com. 
-Dashboard serves two main purposes:
+```
+-l, --language arg
+```
 
-1. Allow users to follow up historical usage data of the service, separated by projects or as a whole.
-2. Allow users to retrieve client_id and client_secret credentials necessary to integrate with Speech Center, please look at authentication section to learn more about how this is performed. 
+Language to use for the recognition: `en-US`, `en-GB`, `pt-BR`, `es`, `es-419`, `tr`, `ja`, `fr`, `fr-CA`, `de`, `it` (default: `en-US`). Please check the [Speech Center streaming Soeech-To-Text documentation](https://speechcenter.verbio.com/documentation/index.php/speech-to-text-streaming/) for an updated list of language codes.
 
+#### Diarization
 
-### Customer Credentials
+```
+-d, --diarization
+```
 
-All speech requests sent to Speech Center must have a valid authorization token in order for the system to successfully process the request. These tokens have a validity period of 1 hour and can be generated on demand using your customer credentials.
+This option enables diarization.
 
-Your customer credentials can be retrieved through the [Speech Center Dashboard](https://dashboard.speechcenter.verbio.com) by logging in with the customer account supplied to you by Verbio.
+> This option is oriented towards batch transcription only and its use for streaming and call automation is not recommended.
 
-#### Authentication flow
 
+#### Formatting
 
-To acquire a valid token submit an HTTP POST request to the authentication service at `https://auth.speechcenter.verbio.com:444`.
+```
+-f, --formatting
+```
 
-**Token expiration management**
+This option will enable formatting on the speech transcription. Please check the [Speech Center streaming Soeech-To-Text documentation](https://speechcenter.verbio.com/documentation/index.php/speech-to-text-streaming/) for an updated list of languages that accept formatting.
 
-![Authentication flow](img/OAuth_diagram.png)
+#### ASR version
 
-As part of the JWT specification, we fully support the token expiration claim, so generated tokens will be valid for only a finite period of time of 1 hour to up to 1 day. It is the responsibility of the calling client to manage this token expiration time and, in a best case scenario, anticipate the refresh by a couple of minutes so the streaming session attempt never fails because of token expiration.
+```
+-A, --asr-version arg
+```
 
-In order to refresh the token, the token refresh endpoint can be called with the same client_id and client_secret, and it will respond with a new JWT token with a renewed expiration time.
+This will select the asr version the speech center will use for transcriptions.
 
+> Please follow Verbio's sales department recommendation on which version to use.
```

Comment:
Here we are saying asr-version but ASR-CSR-STT all three refer to the same thing, so, shouldn't we use STT everywhere? At lest in the doc.

Reply:
Yes, I will standarize.